src/Diagnostics.RuntimeHost/Diagnostics.RuntimeHost.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Was this change intended?

Reply:
Yes. The packages needed to use the log analytics api require 2.1. The build fails with 2.2

Reply:
Is there any workaround? Not sure what the impact of 2.1 would be and moreover we are trying to progress to latest versions (3.x).

Reply:
I also had to do this change on my local proj to get the runtimehost working. @DarrelDonald - lets not push this change for now and keep it only on your box.